base/server/python/pki/server/deployment/__init__.py

```diff
@@ -1314,6 +1314,9 @@ def configure_ca(self, subsystem):
         if ocsp_uri:
             subsystem.set_config('ca.defaultOcspUri', ocsp_uri)
 
+        if config.str2bool(self.mdict['pki_client_verify_cert']):
+            subsystem.set_config('ca.clientRevocationCheck', 'true')
```

Comment:
Could we use something like `ca.certRevocationCheck` instead to avoid confusion, or maybe move it to the connector configuration (e.g. `ca.connector.KRA.certRevocationCheck`)?